src/nlp/features.py

```diff
@@ -400,10 +400,17 @@ def encode_nested_example(schema, obj):
     elif isinstance(schema, Sequence):
         # We allow to reverse list of dict => dict of list for compatiblity with tfds
         if isinstance(schema.feature, dict):
-            return dict(
-                (k, [encode_nested_example(sub_schema, o) for o in sub_obj])
-                for k, (sub_schema, sub_obj) in utils.zip_dict(schema.feature, obj)
-            )
+            if isinstance(obj, (list, tuple)):
```

Comment:
@thomwolf - would be nice if you can take a look here. 
qa4mre is a very special case and I had to make some changes here to make the dataset work. For `qa4mre`, `obj` is a list of `dicts` and so need to be treaded differently.